players/templates/players/watch.html

```diff
@@ -15,17 +15,17 @@
 <script type="text/javascript">
 var GAME_URL_BASE = "{{ game_url_base }}";
 var GAME_URL_PATH = "{{ game_url_path }}";
+var GAME_URL_PORT = {{ game_url_port }};
+var GAME_SSL_FLAG = "{{ game_ssl_flag }}";
 
-// Send host and port to Unity.
-var path = GAME_URL_PATH;
-var proc = (GAME_URL_BASE.substr("http://".length)).split(":");
-var host = proc[0];
 
+// Send host and port to Unity.
 function SendAllConnect() {
-  console.log(`Sending Messages to ${host}${path}`);
-  gameInstance.SendMessage("World Controller", "SetGameURL", host);
-  gameInstance.SendMessage("World Controller", "SetGamePort", 0);
-  gameInstance.SendMessage("World Controller", "SetGamePath", path);
+  console.log(`Sending Messages to ${GAME_URL_BASE}${GAME_URL_PATH}`);
```

Comment:
this log message could be confusing as it contains partial information (only the url base and the path) we need to have a better log message here